components/clm/src/biogeophys/SoilStateType.F90

```diff
@@ -881,11 +881,13 @@ subroutine Restart(this, bounds, ncid, flag)
             dim1name='column', dim2name='levgrnd', switchdim=.true., &
             long_name='soil matric potential', units='mm', &
             interpinic_flag='interp', readvar=readvar, data=this%smp_l_col)
+       if(flag=='read' .and. .not.readvar) this%smp_l_col = -2.5e3_r8
```

Comment:
What is this number?   Add a comment explaining why you are setting this variable to that number.